os/binfmt/binfmt_loadbinary.c

```diff
@@ -224,6 +224,14 @@ int load_binary(int binary_idx, FAR const char *filename, load_attr_t *load_attr
 #ifdef CONFIG_SAVE_BIN_SECTION_ADDR
 	elf_save_bin_section_addr(bin);
 #endif
+
+	/* Print Binary section address & size details */
+
+	binfo("[%s]   text_addr   : 0x%x  size : %u\n", bin->bin_name, (uint32_t)bin->alloc[ALLOC_TEXT], bin->textsize);
+	binfo("[%s]   rodata_addr : 0x%x  size : %u\n", bin->bin_name, (uint32_t)bin->alloc[ALLOC_DATA], bin->rosize);
```

Comment:
can you can maintain same format to print address or size in this PR

Reply:
please check now